src/cocotb_bus/bus.py

```diff
@@ -76,6 +80,16 @@ def __init__(self, entity, name, signals, optional_signals=[], bus_separator="_"
                                         "%s on bus %s" % (sig_name, name))
 
     def _caseInsensGetattr(self, obj, attr):
+        # dir breaks verilator, so avoid calling it if possible
+        for a in (attr, attr.upper(), attr.lower()):
+            if hasattr(obj, a):
+                return getattr(obj, a)
+        if cocotb.SIM_NAME.lower().startswith("verilator"):
+            warnings.warn(
+                "Using dir() for case-insensitive matching;"
+                "this may trigger a known Verilator bug",
```

Comment:
```suggestion
                "Using dir() for case-insensitive matching; "
                "this may trigger a known Verilator bug "
                "(see https://github.com/cocotb/cocotb-bus/pull/56)",
```